debian/control

```diff
@@ -10,7 +10,7 @@ Build-Depends: debhelper (>= 10),
                libopenmpi-dev,
                libssl-dev,
                libyaml-dev,
-               libmercury-dev (= 2.0.0~rc1-1),
+               libmercury-dev (= 2.0.1~rc1-1),
```

Comment:
is there no change log for this ?

Reply:
Yeah, there should be.

Reply:
added new one, though i thought previous entry would be sufficient